parse.go

```diff
@@ -141,6 +144,18 @@ func WithEnvVarPrefix(prefix string) Option {
 	}
 }
 
+// WithEnvVarNoPrefix tells parse to look in the environment for variables with
+// no prefix.  Flag names are converted to environment variables by
+// capitalizing them, and replacing separator characters like periods or hyphens
+// with underscores. Additionally, if the environment variable's value contains
+// commas, each comma-delimited token is treated as a separate instance of the
+// associated flag name.
```

Comment:
Rather than duplicating the explanation text, I think it's OK to say something like "See WithEnvVarPrefix for an explanation of how flag names are converted to environment variables names."

Reply:
Can do.

Reply:
Replaced.